src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -44,15 +44,19 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
       ? chatSettings.recipientId
       : undefined;
 
-  const containerStyles = classNames("chat-sidebar", {
+  const containerStyles = classNames({
+    "chat-sidebar": true,
     "chat-sidebar--expanded": isExpanded,
+    "chat-sidebar--collapsed": !isExpanded,
   });
 
-  const venueChatTabStyles = classNames("chat-sidebar__tab", {
+  const venueChatTabStyles = classNames({
+    "chat-sidebar__tab": true,
     "chat-sidebar__tab--selected": isVenueChat,
   });
 
-  const privateChatTabStyles = classNames("chat-sidebar__tab", {
+  const privateChatTabStyles = classNames({
+    "chat-sidebar__tab": true,
     "chat-sidebar__tab--selected": isPrivateChat,
   });
```

Comment:
NOTE: Simpler to diff/read  when `classNames`' arguments aren't mixed but only strings or only object is used.

Reply:
@mike-lvov @sunny-viktoryia For or against, changing the `classNames` syntax to this one.
Old one: 
```tsx
const privateChatTabStyles = classNames("chat-sidebar__tab", {
    "chat-sidebar__tab--selected": isPrivateChat,
});
```

New one:
```tsx
const privateChatTabStyles = classNames({
    "chat-sidebar__tab": true,
    "chat-sidebar__tab--selected": isPrivateChat,
  });
```

Reply:
ooops, sorry, haven't noticed the comment. Thanks for mentioning me! ❤️ 

I'd prefer the old syntax. For me, it's easier to find what class names are added no matter what. But in the new syntax style, you'll explicitly need to find if there is a class name key with `true` value. what do you think?